src/module.js

```diff
@@ -0,0 +1,8 @@
+// Function to handle the "Clear all completed" button
+export default function setupClearButton(tasks, updateIndexes, saveAndRender) {
+  tasks = tasks.filter((task) => !task.completed);
+  updateIndexes();
+  saveAndRender();
+
+  return tasks; // Return the updated tasks array
+}
```

Comment:
Hey @Pi1998, I think you misunderstood the comment of the previous reviewer. You actually need to move the code responsible for updating the status of a task ie., marking the task as completed or not into a new file. However, as I see, this code is still in the `index.js` file. Kindly move this code to a new file and or in this file and then import it in the `index.js`.

- Optional
- I also noticed, I had to press the **Clear all Completed** button twice every time to clear the tasks marked as complete. Kindly look into it.